letsencrypt/cli.py

```diff
@@ -466,8 +466,9 @@ def auth(args, config, plugins):
 
     # This is a special case; cert and chain are simply saved
     if args.csr is not None:
+        csr = read_file(args.csr)
```

Comment:
While I agree with `csr_path, csr_contents = read_file(...) `, I am not sure if the exception should be caught. In almost all cases, if we don't have these files we cannot continue and have to exit with an error. We could wrap these errors in our own error type or have the verb functions return the error message, but printing an error message and exiting is the proper behavior here.